ops/model.py

```diff
@@ -2713,6 +2713,7 @@ def exec(
 
         Raises:
             ExecError: if the command exits with a non-zero exit code.
+            TimeoutError: if the maximum timeout is reached.
```

Comment:
There was a conversation here, but I can't find it in the GitHub UI any more (because of the rebase?), although it shows up in my email notifications so I don't think I was imagining it :laughing:.

@benhoyt and I had previously discussed this and decided against putting `Raises TimeoutError` in these methods, because they really need to go in all of them, so there's the note in the class docstring instead.

So my inclination is to not add these (and reject the issue asking for it, showing that it's there in the class docstring). Not sure whether @benhoyt agreed with that or not, or whether @IronCore864 you wanted to push back on that?

Reply:
Ah yes, that's right -- I agree we should just remove these "Raises TimeoutError" everywhere and have them in that one place.

Reply:
Got it, I removed them from the methods in `pebble.Client` and `model.Container`; both class docstring have it already.